test/include/armhf_emulation.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
```

Comment:
This file provides setup and teardown functionality for arm emulation that can be used/included in a test script. I.e. it is included and functions are called from it. 
I think it is good enough like this, the file isolates the functionality, it is located in an include directory under the 'test' root folder.

Reply:
well for one, the overlay is not only for the ARM_EMU, i think other parts are using it as well, so that is now more generic.

Further more, it is also setting up /proc, /dev, /etc/mtab and /dev which are unrelated to the arm emulation. So it does the entire setup/teardown, not just the arm emulation specific bits

Reply:
Hmmm.. ok maybe ARM_EMU naming is not nailing it. But still all the stuff done 'except the dev remounting for Docker' is done to create a virtual arm environment where the scripts can be run in. 
So therefore will rename it to...... chroot_env.sh  [dr evil pinky](https://www.google.nl/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=2ahUKEwjzmujSzMHdAhWSaVAKHTYIAKUQjRx6BAgBEAU&url=https%3A%2F%2Ftenor.com%2Fsearch%2Fdr-evil-pinky-gifs&psig=AOvVaw0_QyQZ2im8BHYOUB6Hs8lH&ust=1537258589571672)